lilypond/README.md

```diff
@@ -30,11 +30,12 @@ sudo ./lilypond-2.16.2-1.linux-64.sh
 ```
 
 ### Installation for Windows
-[Read Windows Installation on the Lilypond webpage](http://www.lilypond.org/windows.html)
+[Read Windows Installation on the Lilypond webpage](https://lilypond.org/doc/v2.23/Documentation/web/windows)
 
 Download the `.exe` file and run it to launch the installation process.
 
 ### Installation for Mac OS
+[Read Windows Installation on the Lilypond webpage](https://lilypond.org/doc/v2.23/Documentation/web/macos-x)
```

Comment:
I recommend pointing here instead: https://lilypond.org/download.html

Alternatively, you might restructure the language to say "Lilypond may be downloaded for Windows, Mac, and GNU/Linux. Complete instructions are published on their website: I recommend pointing here instead: https://lilypond.org/download.html